sonar-plugin-api/src/main/java/org/sonar/api/rule/RuleKey.java

```diff
@@ -77,8 +82,12 @@ public String rule() {
     return rule;
   }
 
+  /**
+   * @deprecated since 5.5, manual rule feature has been dropped
+   */
+  @Deprecated
   public boolean isManual() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)